### PR TITLE
guides: Introduction section review

### PR DESCRIPTION
**Pull request by milmazz**
This PR fixes a few things in the "Introduction" guides:

* Add the *Authentication* item to the _Overview page_ to match the listed sections in the sidebar

image

* Replace the **Channels** _group section_ with _Real-Time_, to match the Overview list description.

image

* Replace the "Introduction" title to be "Channels", which is located under the "Real-Time" section

image
 
* Remove the un-existent output _Webpack is watching the files_ from the "Up and Running" page.

### CODE REVIEW

File: guides/channels/channels.md
Comment (milmazz) [inline guides/channels/channels.md:1]: Should I update the file path from `guides/channels/channels.md` to be something like `guides/real-time/channels.md`? Just to follow the pattern described in the sidebar.

**Review by cw789 — APPROVED** [review]

File: guides/channels/channels.md
Comment (cw789) [reply guides/channels/channels.md:1]: I would do so. As it's easier to find `presence.md` then.
But wait for some of the maintainers to approve this oppinion.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by josevalim** [comment]
Feel free to send a PR to rename the directory too!

**Pull request merged**
